clio-server/src/main/scala/org/broadinstitute/clio/server/ClioServerConfig.scala

```diff
@@ -1,48 +1,63 @@
 package org.broadinstitute.clio.server
 
-import org.broadinstitute.clio.util.model.Env
+import org.broadinstitute.clio.util.json.ModelAutoDerivation
+import org.broadinstitute.clio.util.model.ServiceAccount
 
 import com.typesafe.config.Config
 import net.ceedubs.ficus.Ficus._
 import net.ceedubs.ficus.readers.ArbitraryTypeReader._
 import net.ceedubs.ficus.readers.ValueReader
 
+import scala.collection.JavaConverters._
 import scala.concurrent.duration.FiniteDuration
 
+import java.io.File
+import java.nio.file.{Files, Path}
+
 /**
   * Configuration file accessors.
   */
 object ClioServerConfig {
-  private val config = ClioConfig.getConfig("server")
+  private val server = ClioConfig.getConfig("server")
```

Comment:
Why rename this? The object seems more like `config` than `server` to me. Would `serverConfig` be better?

Reply:
Will use `serverConfig`. I renamed it because `ModelAutoDerivation` has a member named `config`, so I was getting name clashes.